### DISCUSSION TITLE
transfer subscriptions

### DISCUSSION THREAD

**Discussion by knightmar** [FreeTubeApp/FreeTube#3239] [Q&A]
hello, I was wondering if it was possible to transfer your subscriptions from your google account to Libretube?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @knightmar are u sure you are in the right repo? Your question is about [LibreTube](https://github.com/libre-tube/LibreTube).

You are now in the **FreeTube** repo.

**Reply by knightmar** [reply]
yeah, sorry 😔😔.
I was using LibreTube before but want to switch on FreeTube, sorry for the mistake

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
No problem. See https://docs.freetubeapp.io/usage/importing-subscriptions/ to import your subscriptions :)
